tools/wasm-sample/src/main.rs

```diff
@@ -0,0 +1,11 @@
+fn main() {
```

Comment:
May I ask you to update or create the docs for this tool? How can I use it?

Reply:
Added [README](https://github.com/containers/youki/blob/84ab1655ae49c1bc09ccd1c266cd811f58ca9998/tools/wasm-sample/README.md)

Reply:
Thanks :heart_eyes: